src/commands/generate/inspec_profile.ts

```diff
@@ -11,12 +11,12 @@ import {Logger} from 'winston'
 import _ from 'lodash'
 
 export default class InspecProfile extends BaseCommand<typeof InspecProfile> {
-  static readonly usage =
-    '<%= command.id %> -X <[stig or cis]-xccdf-xml> [--interactive] [-L info|warn|debug|verbose] ' +
-    '[-m <metadata-json>] [-s] [-T rule|group|cis|version] [-O <oval-xccdf-xml>] [-o <output-folder>]'
+  static readonly usage
+    = '<%= command.id %> -X <[stig or cis]-xccdf-xml> [--interactive] [-L info|warn|debug|verbose] '
```

Comment:
Should we make an issue in the repo to add the same url functionality to this command? No need to put it in this PR.

Reply:
That one is a bit more tricky as I'm not sure how a cis zip package looks like.